tests/pjit_test.py

```diff
@@ -4113,6 +4113,22 @@ def f():
     with mesh:
       f()  # doesn't crash
 
+  def test_lowering_cache_hit_with_closed_over_constants(self):
+    np_inp = np.arange(8)
+    arr = jnp.arange(8)
+    @jax.jit
+    def f(x):
+      return x + np_inp
+
+    with (jtu.count_pjit_cpp_cache_miss() as cpp_count,
+          jtu.count_jit_tracing_cache_miss() as tracing_count,
+          jtu.count_jit_and_pmap_lowerings() as lowering_count):
+      f(np_inp); f(np_inp)
```

Comment:
put them all on newlines please :) (this is a test lol, no need to save lines here ;))